deploy/org.mavlink.qgroundcontrol.metainfo.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.mavlink.qgroundcontrol</id>
+  
+  <name>QGroundControl</name>
+  <summary>UAS ground control station</summary>
+  
+  <metadata_license>MIT</metadata_license>
+  <project_license>GPL-3.0-only</project_license>
```

Comment:
And here only one of the licenses. Don't know whether mixes are possible within these metadata files.

Reply:
Found a solution to declare GPL3 and Apache2.